apis/filesystem/src/main/java/org/jclouds/filesystem/strategy/internal/FilesystemStorageStrategyImpl.java

```diff
@@ -154,32 +161,95 @@ public boolean createContainerInLocation(String container, Location location, Cr
    }
 
    @Override
-   public ContainerAccess getContainerAccess(String container) {
+   public synchronized ContainerAccess getContainerAccess(String container) {
       Path path = new File(buildPathStartingFromBaseDir(container)).toPath();
-      Set<PosixFilePermission> permissions;
-      try {
-         permissions = getPosixFilePermissions(path);
-      } catch (IOException ioe) {
-         throw Throwables.propagate(ioe);
+
+      if ( System.getProperty("os.name", "").toLowerCase().contains("windows") ) {
```

Comment:
Extract this to a method in the [TestUtils](https://github.com/jclouds/jclouds/blob/master/core/src/test/java/org/jclouds/utils/TestUtils.java) class, similar to what we already do with OSX.